src/blueprint.zig

```diff
@@ -21,6 +21,7 @@ const BinaryWriter = main.utils.BinaryWriter;
 const BinaryReader = main.utils.BinaryReader;
 
 pub const blueprintVersion = 0;
+var voidTyp: ?u16 = null;
```

Comment:
`typ` is only used because of conflicts with the keyword
```suggestion
var voidType: ?u16 = null;
```